.github/workflows/docs.yml

```diff
@@ -2,8 +2,10 @@ name: Deploy mdbook documentation
 
 on:
   push:
-    branches:
-      - main
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
Do you really want this deployed on each PR?

Reply:
The deploy step is conditional on this running off the main branch, so it would be skipped during PRs.
But I think there's value in *building* the book on each PR.

Reply:
Like here, the actual "Deploy" step is skipped.
https://github.com/containerd/runwasi/actions/runs/13543206506/job/37848751089?pr=881

Reply:
Oops! You are right. I missed the conditional and totally agree with your reasoning. Resolved!